plugins/sip/lib/model/providers/pexip/kPexipUtils.php

```diff
@@ -103,48 +105,55 @@ public static function retrieveAndValidateEntryForSipCall($queryParams, $pexipCo
 		if (!$dbLiveEntry)
 		{
 			$msg = "Entry was not found for sip token $sipToken";
+			self::sendSipEmailNotification($dbLiveEntry->getPartnerId(), $dbLiveEntry->getCreatorPuserId(), $msg, $dbLiveEntry->getId());
 			KalturaLog::err($msg);
```

Comment:
In all the following cases , err with backtrace is not required , It enough to  print Notice or Warning.
Applies to the entire page here